test/e2e/common.go

```diff
@@ -181,6 +182,9 @@ func DownloadFile(filePath string, url string) error {
 	if err != nil {
 		return err
 	}
+	if resp.StatusCode != http.StatusOK {
+		return errors.Errorf("failed to download image from %s got %d", filePath, resp.StatusCode)
```

Comment:
```suggestion
		return errors.Errorf("failed to download image from %q got %q", filePath, http.StatusText(resp.StatusCode))
```
Maybe quote the outputs, and unpack the status code to proper status text, ie `404` => `404 Not Found` ?

Reply:
Good suggestion @tuminoid , +1

Reply:
done